cards/src/main/resources/cards/custom/group0/other_death_knight/koft/hero_arthas_menethil.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Arthas Menethil",
+  "heroPower": "hero_power_fallen_champions",
+  "baseManaCost": 10,
+  "type": "HERO",
+  "heroClass": "SPIRIT",
+  "rarity": "LEGENDARY",
+  "description": "Battlecry: Freeze all characters and decks.",
```

Comment:
I agree, and I think the best way to do it then actually would be countUntilCast of 3 for ALL TurnEndTriggers, so that way it would stay for each players next full turn no matter who applies it (this changing being made in the enchantment itself)